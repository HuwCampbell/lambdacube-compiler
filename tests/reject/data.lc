data Data1 = Data1 a b c
