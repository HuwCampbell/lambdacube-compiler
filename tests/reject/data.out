can't find: a in "./tests/reject/data.lc" (line 1, column 20):
data Data1 = Data1 a b c
                   ^